### ISSUE TITLE
[Publisher][Design Update] Agency Settings Design Update (1/5)

### ISSUE THREAD

**Issue by mxosman** [Recidiviz/justice-counts#1254]
Create a feature branch that will house all of the commits for this feature. Once the feature is ready to launch, this will be the branch that will be merged into `main`.

**Comment by corypride** [comment]
Feature branch: 'cory/agency-settings-design-updates' created and pushed to the repo.

**Comment by mxosman** [comment]
Thank you, @corypride!
